ui/css/treeherder-info-panel.css

```diff
@@ -11,6 +12,21 @@ div#info-panel {
   flex-flow: column;
 }
 
+.info-panel-slide {
+  animation: info-panel-slide 0.4s;
+  -webkit-animation: info-panel-slide 0.4s;
+}
+
+@keyframes info-panel-slide {
+  0% { transform: translateY(100%); }
+  100% { transform: translateY(0%); }
+}
+
+@-webkit-keyframes info-panel-slide {
```

Comment:
We have this smattered around the repo also (treeherder-pinboard, perf):
https://github.com/mozilla/treeherder/search?utf8=%E2%9C%93&q=-webkit-keyframes&type=Code

Historically Treeherder has notionally supported Chrome, in addition to FF. If we should pull it in those spots also, or if -webkit is now redundant for Chrome support, let me know. I was going based of the latest MDN here:

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations#Examples

Thanks for the feedback @darkwing !